test/unit/models/flag.test.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable security/detect-object-injection */
+const chai = require("chai");
+const { expect } = chai;
+
+const cleanDb = require("../../utils/cleanDb");
+
+const flagModel = require("../../../models/flag");
+const firestore = require("../../../utils/firestore");
+const flagSnapshot = firestore.collection("featureFlags");
+const flagData = require("../../fixtures/flag/flag")();
+
+describe("flags", function () {
+  let flagId;
+
+  beforeEach(async function () {
+    flagId = await flagModel.addFlag(flagData);
+  });
+
+  afterEach(async function () {
+    await cleanDb();
+  });
+
+  describe("add feature flag", function () {
```

Comment:
Is the module name correct? I feel we are not checking for just "add feature flag" in this scenario, but more of whether feature flags are working correctly or not.

Reply:
But we are checking whether the process of adding feature flag was done correctly or not so it becomes a suitable name i guess. Do you have any other suggesstions.

Reply:
In all other test cases, we are writing the function name here and not what it does. This makes it easier to find for what function the test cases are running. So ideally it should be `addFeatureFlag`. What do you think?